[OSDOCS#17058]: DC-2: Image Mirroring and oc mirror Tooling-CQA-2 (#115637)
openshift/openshift-docs: modules/mirror-registry-localhost-update.adoc

[id="mirror-registry-localhost-update_{context}"]
= Updating mirror registry for Red Hat OpenShift from a local host

[role="_abstract"]

> This procedure explains how to update the _mirror registry for Red{nbsp}Hat OpenShift_ from a local host by using the `upgrade` command. Updating to the latest version ensures new features, bug fixes, and security vulnerability fixes.

Reviewer: Slight edit to avoid self-referential language:

```suggestion
You can update the _mirror registry for Red{nbsp}Hat OpenShift_ from a local host by using the `upgrade` command. Updating to the latest version ensures new features, bug fixes, and security vulnerability fixes.
```
